app/server/static/package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
       "dev": true,
       "requires": {
-        "@babel/highlight": "7.0.0"
+        "@babel/highlight": "^7.0.0"
```

Comment:
Given that `package.json` didn't change as part of this pull request, can we revert the changes to `package-lock.json` since they don't pertain to the changes in this pull request?

Reply:
Agreed. However, I just git merged with master branch, and it has `"@babel/highlight": "^7.0.0"` now, so I left it